### PR TITLE
pagination is ready

### PR DESCRIPTION
**Pull request by maftunabonu**
pagination is ready. There is a huge space between rows, though

### CODE REVIEW

**Comment by Sirojiddin96** [comment]
Dont forget to check reviewers please.

File: src/Pages/Products/Products.js [hunk 1]
```diff
@@ -14,37 +14,64 @@ import LineCardIconBlue from "../../assets/icons/LineCardIconBlue.svg";
...
+        else if(option === "Price min-max"){
+            setSort([...products].sort((a, b)=> {
+                return a.originalPrice - b.originalPrice;
+            }))
+            }
+        else{
         setSort([...products].sort((a, b) => {
-            return a.originalPrice - b.originalPrice;
+            return b.originalPrice - a.originalPrice;
         } ) )
```
Comment (Sirojiddin96) [inline src/Pages/Products/Products.js:48]:
```
const handleSort=(option)=>{
const categorType=option.target.value
 const sortedProducts=products.filter(prod=>prod.type==categorType)
     .sort((a, b)=> a.id - b.id;
setSort(sortedProducts)
}
```

How about this one?
categoryType and prod.type need to matched, Please match object key to value
IF you face any late state update, you can use 
useEffect(()=>{})

Comment (maftunabonu) [reply src/Pages/Products/Products.js:48]: It does not seem to be working

Comment (Sirojiddin96) [reply src/Pages/Products/Products.js:48]: let me check it after a while. If everything works, you can merge, we can refactor functions later.

**Review by Sirojiddin96 — APPROVED** [review]

**Comment by Sirojiddin96** [comment]
For Description page, 
New Branch needs to be created :)
You need to squash and merge this branch after finish your work.

**Comment by Sirojiddin96** [comment]
Additionally, delete branch you merged srazi. :)

**Comment by Sirojiddin96** [comment]
You are doing really good job. Honestly, You are learning faster than I thought.

**Pull request merged**